canonical-web-and-design/maas-ui
# feat(ui): separate AddKVM form into LXD and virsh-specific forms

## Done

- Removed the component that wrapped `AddLxd` and `AddVirsh` so that they're fully independent

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the LXD KVM list and check that the button in the top right says "Add LXD host"
- Click it and check that the Add LXD host form appears without a radio button for changing to virsh
- Check the same in the Virsh KVM list

## Fixes

Fixes canonical-web-and-design/app-squad#338
Fixes canonical-web-and-design/app-squad#339
